src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -9171,11 +9171,14 @@ let isReferenceTyparTy g ty =
     | ValueNone ->
         false
 
-let isSupportsNullTyparTy g ty = 
+let IsSupportsNullTyparTy g ty = 
```

Comment:
Nit: for extra clarity this should probably be renamed to something starting with `Get` or `TryGet`. `GetTyparTyIfSupportsNull`?

And maybe keep `IsSupportsNullTyparTy g ty = (GetTyparTyIfSupportsNull g ty).IsSome`